python/tvm/relay/op/transform.py

```diff
@@ -214,6 +213,28 @@ def reshape(data, newshape):
         newshape = [newshape]
     return _make.reshape(data, list(newshape))
 
+def argwhere(condition):
```

Comment:
Can `argwhere` takes Tensors in dtype other than bool? If so, we should add test example. Otherwise, we should check the dtype in the typerel.

Reply:
an example in tf:
```
condition = [[100, 0, -1], [0, 1, 2]]
with tf.Session() as sess:
  print(sess.run(tf.where(condition)))

-- output:
[[0 0]
 [0 2]
 [1 1]
 [1 2]]
```

Reply:
Thanks. I extended it to support more dtypes and added unit test.